### PR TITLE
Issue #14 Build JFixture-Scala against both Scala 2.11 and 2.12

### PR DESCRIPTION
**Pull request by Rocketeer007**
Add `build.sbt` for cross compiling using sbt.

Update `README.md` to show syntax for sbt and for maven using latest version

### CODE REVIEW

**Comment by Rocketeer007** [comment]
I'm not 100% certain about three points here:
1. Should I remove the Maven `pom.xml` file as part of switching to SBT? 
1a. Will doing that break the Travis build? It seems to start with `mvn install`, then run `build.sh`...
2. Have I broken the publishing logic which gets the security details from the Travis environment?
3. Is `1.0.1-SNAPSHOT` the right version for this?

File: build.sbt [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+ThisBuild / organizationHomepage := Some(url("http://www.flextrade.com/"))
+ThisBuild / description  := "JFixture-Scala adds support for scala types and collections to JFixture"
+ThisBuild / homepage     := Some(url("https://github.com/FlexTradeUKLtd/jfixture-scala"))
+
+ThisBuild / startYear    := Some(2008)
+ThisBuild / licenses     += "MIT License" -> url("http://www.opensource.org/licenses/mit-license.php")
+
+ThisBuild / developers := List(
+  Developer(
+    id = "albx79", name = "Alberto Colombo", email = "albx79@gmail.com", url = url("https://github.com/albx79")
```
Comment (phutchin) [inline build.sbt]: Can you change this to 
<email>opensource@flextrade.com</email>
            <organization>FlexTrade UK Ltd</organization>
            <organizationUrl>http://www.flextrade.com/</organizationUrl>

**Comment by phutchin** [comment]
Actually just saw your questions:


1. Yes we should remove the pom.   
2. It doesn't look broken, but we can go through some tests to make sure it works after this, and if need be push another fix.
3. Master should probably be on 1.1.0-SNAPSHOT (looks like whoever released last didn't bump it), so then this would be in a 1.1.0 release

File: build.sbt (hunk 1 above)
Comment (Rocketeer007) [reply build.sbt]: Okay, done

**Pull request merged**